src/client/triggerUpdate.js

```diff
@@ -0,0 +1,14 @@
+import batchUpdate from './batchUpdate'
+
+// store an id to keep track of DOM updates
+let batchId = null
+
+export default function triggerUpdate(vm, hookName) {
+  if (vm.$root._vueMetaInitialized && !vm.$root._vueMetaPaused) {
```

Comment:
What do you think about grouping these properties in a single `_vuemeta` object? Then the `_vuemeta` property will be an object on `$root` and a boolean on all childNodes of $root (so it also replaced `_hasMetaInfo`, but that could be a breaking change althought its only  meant to be used internally)

Reply:
Like the idea, yup!

Reply:
About the breaking change, that's true. And might influence things like `nuxt-i18n`. We should def. point that out

Reply:
Wdyt about this solution, now I provide a `hasMetaInfo()` export which external libraries could and should use to check if meta info has been defined. That way we can keep the exact implementation for that internally (like _hasMetaInfo was meant to be)

Reply:
Hmm, this works for the es module but not for cjs or umd due to combining default and named exports ofc

Reply:
Would you be ok with removing the single / default export and only have named exports? Its a good reason to release 2.0 ;)

Reply:
@pimlie I'd be fine with that!

Reply:
I changed it to that but didnt like it. So now I just added a default object export. Looked a bit around and this seems to be the more used solution by others vue plugins as well (they either do this or export a class with a static install method, the latter we might use for 3.0)